tests/models/t5/test_modeling_t5.py

```diff
@@ -1455,6 +1487,7 @@ def test_summarization(self):
             [model.config.prefix + x for x in [FRANCE_ARTICLE, SHORTER_ARTICLE, IRAN_ARTICLE, ARTICLE_SUBWAY]],
             padding="max_length",
             truncation=True,
+            max_length=512,
```

Comment:
nit, otherwise we can't form a tensor. Apparently it stopped truncating without us passing the `max_length`